cmd/cli/kubectl-kyverno/utils/store/store.go

```diff
@@ -5,29 +5,49 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 )
 
+type Context struct {
+	Policies []Policy `json:"policies"`
+}
+
+type Policy struct {
+	Name  string `json:"name"`
+	Rules []Rule `json:"rules"`
+}
+
+type Rule struct {
+	Name          string                   `json:"name"`
+	Values        map[string]interface{}   `json:"values"`
+	ForEachValues map[string][]interface{} `json:"foreachValues"`
+}
+
+type Subject struct {
+	Subject rbacv1.Subject `json:"subject,omitempty" yaml:"subject,omitempty"`
+}
+
 var (
-	Mock           bool
+	mock           bool
 	registryClient registryclient.Client
-	AllowApiCalls  bool
-	ContextVar     Context
-	ForeachElement int
-	Subjects       Subject
+	allowApiCalls  bool
+	policies       []Policy
+	// contextVar     Context
+	foreachElement int
+	subject        rbacv1.Subject
```

Comment:
Should we make these a struct? Something like storeOptions?

Reply:
we don't care, the store itself is not a struct.